### PR TITLE
Edit/533 get started

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#533 

### Please describe the goal of this pull request and the changes that were made.

Update Get Started for V1 docs 

### Additional Comments

Questions/Comment for @geealbers 

- Removed Python 2 tip because I'm hoping that will be resolved with 11ty
- I will likely need to update "The project is ready when you see the message: “Theme and dependencies successfully installed" once we know how 11ty cli works.
- I also will need to go back and confirm all Quire commands will work the same
- I toned down explicit mentions of GitHub, including removing the tip about serial publications. Aside from the overt GitHubness of it all, I'm not sure the tip belongs in this section. Thoughts? 

I'm not sure if how I approached breaking down the folder structure make sense:

- It's a little challenging to explain that the content folder includes Markdown files for the project **and** a bunch of other folders, but I assume there's nothing that can be done about that.
- I dont know how to explain the computed folder
- Does assets still pertain to "anything that doesn't first need to be process through the static site generator" as was the case with the `static` directory where images previously lived. 
- Will I need to add a directory that explains where PDF, site and EPUB downloads go?
- I removed the README but happy to add it back in. Just let me know where you think it belongs (editors v. devs) 
- The styles folder description might need more information.
- Did I highlight the correct folders for developers? (The inclusion of data and assets is a little confusing)
- I don't know how to explain _includes, _layouts, and _plugins. I looked at the READMEs for those folders, but it was too technical for me
- I put Javascript in the dev section; not sure if that's correct. If it is, then that file structure is also a little confusing. 

- Is the way I handled the sample publication outline ok? (including the "not included in outline" note)
- I removed the note about not using index.md because it looks like we do use it in 11ty. I also added it to the outline but realized I don't explain its purpose. Should I do that?
- I looked in the 11ty docs to see how slugs are handled. Is the updated tip correct?
- I added some content to the Text and Images section because it makes sense to include something in between creating an outline and previewing your project, but I didn't want to get too redundant with the other sections I link to.

I think those are all my questions, but open to any other advice on how to improve this section. Explaining the file structure was very challenging.

### CODE REVIEW

File: content/documentation-v1/getting-started.md [hunk 1]
```diff
@@ -5,146 +5,134 @@ weight: 6080
...
+### 📁 _data
 
-### 📁 config
+This directory also contains the `config.yaml` file which is used expressly for configuring how Quire operates and for defining a number of key values used in Quire {{< q-def "templates" >}}. Users who have worked on other non-Quire projects will note that they typically use the `config.yaml` file to also store publication metadata. Given the potentially large scope of this metadata in formal digital publications, Quire uses the `publication.yaml` file inside the `_data` directory instead. Read more in [*Metadata & Configuration*](/documentation/metadata-configuration/).
```
Comment (geealbers) [inline content/documentation-v1/getting-started.md:109]: I might say something like the following, the rest of the paragraph isn't true for 11ty like it was for Hugo. And we should revisit this when the team finishes refactoring the config.yaml file (https://jira.getty.edu/browse/DEV-12846).

> This directory also contains the `config.yaml` file which is used for configuring different layout options built into Quire, and for defining a number of key values used in Quire {{< q-def "templates" >}}.

File: content/documentation-v1/getting-started.md [hunk 2]
```diff
@@ -5,146 +5,134 @@ weight: 6080
...
+
+
+#### 📁 styles
+
+This directory includes a `variables.scss` file and a `custom.css` file for directly overriding theme styles (read more in [*Style Customization*](/documentation/styles-customization/)).
 
 ## Files for Developers
 
-Inside each Quire project, you will find the following directories and files. Developers will primarily use the `config.yml` file and the `bin`, `config`, `site`, and `theme` directories.
+Developers will primarily use the `_includes`, `_layouts`, `_plugins` directories and the `config.yaml` file.
```
Comment (geealbers) [inline content/documentation-v1/getting-started.md:90]: I actually think `config.yaml` is more geared to the non-technical user. Developer-level configuration would happen other places.

File: content/documentation-v1/getting-started.md [hunk 3]
```diff
@@ -157,43 +145,45 @@ The names of the files will effect the final URLs of your publication. By defaul
...
+## Add Text and Images
 
-## Preview and Edit a Project
+Once an outline has been created, text and images can be added to the project.
 
-Quire lets you preview the current version of your site in a web browser, and will update the preview as you edit the files.
+In many cases, text will originate as a Word document. You can use Pandoc to convert Word documents into Markdown. Instructions can be found in the [YAML & Markdown](/documentation/fundamentals/#microsoft-word-to-markdown-conversion) section of this guide.
 
-**To run the preview:**
+Once images are added to the `_assets/images` folder and entries have been added to the `figures.yaml`, then you can use shortcodes to insert the images into their respective Markdown files. More information can be found in the [Figure Images](/documentation/figure-images/) section of this guide.
```
Comment (geealbers) [inline content/documentation-v1/getting-started.md:169]: I'm not sure if we're consistent, but my instinct is to always follow directory names with a slash: `_assets/images/`. The slash is the equivalent to a file suffix, like `.md`. What do you think?

**Review by geealbers — COMMENTED** [review]
Good start. We'll obviously need to take another pass once we have the CLI working, but good for now. Just a few notes for you.

File: content/documentation-v1/getting-started.md (hunk 3 above)
Comment (Erin-Cecele) [reply content/documentation-v1/getting-started.md:169]: Noted! I've fixed it here but will likely need to correct it elsewhere in this guide and in the Getty Quire Guide.

**Comment by Erin-Cecele** [comment]
@geealbers The requested edits have been made. Should we just leave the PR open while we wait for the various 11ty confirmations?

**Pull request merged**